wechatpy/constants.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import, unicode_literals
 
-from enum import Enum
```

Comment:
how about import [enum.unique](https://cpython-test-docs.readthedocs.io/en/latest/library/enum.html#enum.unique) 

and add this decorator to unique enum class.

for example
```
@enum.unique
class ReimburseStatus(Enum):
    pass
```